release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -578,6 +578,12 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-6-deprecated-features"]
 === Deprecated features
 
+[discrete]
+[id="ocp-4-6-tls-common-name"]
+==== TLS verification falling back to the Common Name field
```

Comment:
@sttts @sdodson Can you review this deprecation notice in the release notes?

Preview: https://bz-1886892--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-deprecated-features

Reply:
lgtm

Reply:
lgtm